sensorhub-core/src/main/java/org/sensorhub/impl/sensor/SensorSystem.java

```diff
@@ -172,10 +169,18 @@ private IModule<?> loadModule(ModuleConfig config)
     protected void handleEvent(Event e)
     {
         if (e instanceof ModuleEvent)
+        {
             eventHandler.publish(e);
+            if(((ModuleEvent) e).getType() == ModuleEvent.Type.CONFIG_CHANGED)
+            {
+                var moduleConfig = ((ModuleEvent)e).getModule().getConfiguration();
+                for(SystemMember member : config.subsystems)
+                    if(moduleConfig.id.equals(member.config.id))
```

Comment:
Since you are iterating over a collection, seeking a specific match, I recommend adding a break statement to this if conditional block such that on a match, the break terminates the loop.  Ex. let's say that a system has N subsystems, if the matching subsystem is the first subsystem to be read from the collection, then the for loop will continue N-1 times each time unnecessarily checking if the id's match for the remaining subsystems.

`if (conidtion met) {
    do work;
    break;  // This break will break out of for loop
}   
`